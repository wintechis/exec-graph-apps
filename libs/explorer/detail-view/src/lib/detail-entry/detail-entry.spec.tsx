import { render } from '@testing-library/react';

import DetailEntry from './detail-entry';

describe('DetailEntry', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<DetailEntry label="A number" value={100} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
