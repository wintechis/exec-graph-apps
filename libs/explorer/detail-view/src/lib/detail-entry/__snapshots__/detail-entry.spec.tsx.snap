// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailEntry should render successfully 1`] = `
<DocumentFragment>
  <div
    class="py-1 sm:grid sm:grid-cols-3 sm:gap-4"
  >
    <dt
      class="font-medium text-gray-500 break-all"
    >
      A number
    </dt>
    <dd
      class="mt-1 text-gray-900 sm:mt-0 sm:col-span-2 break-all"
    >
      100
    </dd>
  </div>
</DocumentFragment>
`;
